composer.json

```diff
@@ -12,6 +12,11 @@
     "config": {
         "platform": {
             "php": "8.0.99"
+        },
+        "allow-plugins": {
```

Comment:
Oh, composer also broke compat here? :| 

I'm starting to see frikken BC breaks everywhere...

Reply:
Wait hold up, we have to allow plugins in every package now?